test/cli/package-bad-imports/package-bad-imports.out

```diff
@@ -0,0 +1,39 @@
+example/__package.rb:5: Cannot find package `NonexistentPackage` https://srb.help/3704
+     5 |  import NonexistentPackage
+                 ^^^^^^^^^^^^^^^^^^
+
+example/__package.rb:7: Cannot find package `DoNot::SUggestThis` https://srb.help/3704
+     7 |  import DoNot::SUggestThis
+                 ^^^^^^^^^^^^^^^^^^
```

Comment:
There's a constant in this test case called `DoNot::SuggestThis` which _isn't_ a package. The fact that we don't suggest it in this test is part of the point: we shouldn't suggest non-package constants.